lib/nationbuilder/client.rb

```diff
@@ -16,6 +16,10 @@ def initialize(nation_name, api_key, opts = {})
     end
   end
 
+  def http_client
```

Comment:
Dependency Injection works just as well. It makes some use cases easier, such as proxies and mocking tests, though if I just want to log certain calls I now need to retain a reference to the client. On balance it's better, I think, though.

Did you want me to change the pull request?